packages/task/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@theia/task",
+  "version": "0.2.3",
+  "description": "Theia - Task extension",
+  "dependencies": {
+    "@theia/core": "^0.2.3",
+    "@theia/markers": "^0.2.3",
+    "@theia/process": "^0.2.3",
+    "@theia/terminal": "^0.2.3",
+    "@theia/workspace": "^0.2.3",
+    "require-strip-json-comments": "^1.0.2"
```

Comment:
I thought `jsonc-parser` ended up being used by modifying the webpack config file? If that is also used it's fine though.

Also if you really use `jsonc-parser`, shouldn't that dependency be specifically mentionned in the task package.json?

Reply:
@epatpol I think you were looking at the previous version? This is no longer in there.

Reply:
> Also if you really use jsonc-parser, shouldn't that dependency be specifically mentionned in the task package.json?
It's added in the latest version